README.md

```diff
@@ -376,11 +392,14 @@ Thus, the possible options are:
 
 - `type`, like the attribute
 - `position`, like the attribute
-- `closeButton`, like the attribute
 - `duration`, like the `show()` option
 - `multiple`, like the `show()` option
 - `newestOnTop`, like the `show()` option
-- `action`, a string or `Element`.
+- `closeButton`, a boolean or string.
```

Comment:
Are we planning on having a `closebutton` option for the `show()` method?

Reply:
Nah, since it's not specific to a single instance of showing (like the timeout is), it should be done as a setup before calling show()